Backpack/CardButton/Classes/SaveCardButton.swift

```diff
@@ -0,0 +1,273 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import UIKit
+
+public class BPKSaveCardButton: UIButton {
+
+    public var checked: Bool {
+        didSet {
+            updateLookAndFeel()
+            onCheckedChange?(checked)
+        }
+    }
+
+    public var style: BPKCardButtonStyle {
+        didSet { updateLookAndFeel() }
+    }
+
+    public var size: BPKCardButtonSize {
+        didSet { updateLookAndFeel() }
+    }
+
+    public var onCheckedChange: ((Bool) -> Void)?
+
+    private let viewConfigurator = CardButtonViewConfigurator()
+    private let containedBackgroundCircle: UIView
+
+    public init(
+        checked: Bool = false,
+        accessibilityLabel: String,
```

Comment:
Not a huge fan of forcing a label in the init, maybe having the docs state it's required? Or was this part of the api table?

Reply:
If we do force the label, could we then have an 'empty string' check and maybe an assertion fail? 
Right now if the label is empty, we have VoiceOver make a guess based on the icon.

We might want to deal with that case.

Reply:
> was this part of the api table?
Yes. The figma table has it. [link](https://www.figma.com/file/irZ3YBx8vOm16ICkAr7mB3/Backpack?node-id=24699%3A29708&t=iBugbhC92rScVCdR-4)

Do we have any example of how to do 'empty string check'?
The only other class with a required `accessibilityLabel` is `BPKButton` for swift-UI

Reply:
Hm, we can leave as-is for now.